lib/engine/game/g_18_ny/game.rb

```diff
@@ -166,6 +212,44 @@ def can_par?(corporation, _parrer)
         def can_hold_above_limit?(_entity)
           true
         end
+
+        # Operating round logic
+
+        def operating_order
+          minors, majors = @corporations.select(&:floated?).sort.partition { |c| c.type == :minor }
```

Comment:
why not just sort by [floated?, type.minor ? 0 : 1]

Reply:
Not sorted on floated?. It is sorted on operating order, which is defined within corporation.

Reply:
ah i see nvm then